### ISSUE TITLE
ParMetis in third_party

### ISSUE THREAD

**Issue by ollielo** [losalamos/flecsi#43]
Are we going to use ParMetis for distributed graph partition for the mesh? If so, can you add ParMetis in the third_party repo? Or should this be an issue the flecsi-third-party repo?

**Comment by junghans** [comment]
This is related to #42, but make it an issue on the losalamos/flecsi-third-party and assign it to @charest.

**Comment by junghans** [comment]
There is already a `FindParMETIS.cmake` in cinch, so do you can use:
```
if(ENABLE_MPI)
  find_package(ParMETIS)
else()
  set(PARMETIS_INCLUDE_DIRS)
  set(PARMETIS_LIBRARIES)
endif()
```

**Comment by ollielo** [comment]
Solved with [427ae5e](https://github.com/losalamos/flecsi/commit/427ae5eaa84e7287028cca4dc598c56ddeb3d8d0).
